rails/react-components/src/library/components/permission-forms-v2/use-fetch.ts

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
```

Comment:
Maybe move this to a shared `hooks` folder under `src/library` (created folder if needed)?

Reply:
Great, done, thanks.